monitoring/logging.sh

```diff
@@ -99,21 +99,22 @@ findAndClearOrphans() {
 # set the name of the asgs log file
 setSyslogFileName()
 {
-   local DATETIME=$(date +'%Y-%h-%d-T%H:%M:%S%z')
+   local DATETIME=$(date +'%D %H:%M:%S')
    if [[ ! -d $WORK/log ]]; then
       mkdir -p $WORK/log
-      echo "[$DATETIME] INFO: Created subdirectory for ASGS log files : '$WORK/log'."
+      echo "[$DATETIME] Created subdirectory for ASGS log files : '$WORK/log'."
    else
-      echo "[$DATETIME] INFO: Found subdirectory for ASGS log files : '$WORK/log'."
+      echo "[$DATETIME] Found subdirectory for ASGS log files : '$WORK/log'."
    fi
    SYSLOG=${SYSLOG:-$WORK/log/${INSTANCENAME}.asgs-${STARTDATETIME}.$$.log}
-   consoleMessage "Set ASGS log file parameter SYSLOG to '$SYSLOG'." $SYSLOG
+   consoleMessage "SYSLOG set to"
+   consoleMessage "$SYSLOG'." $SYSLOG
```

Comment:
why are these separate messages?

Reply:
I did that because `$SYSLOG` can be a very long path. I can look at how to do that without invoking `consoleMessage` twice like that.

Reply:
fixed